src/main/java/seedu/address/model/client/ClientEmail.java

```diff
@@ -50,7 +50,7 @@ public ClientEmail(String email) {
     public static class EmptyEmail extends ClientEmail {
         public static final ClientEmail EMPTY_EMAIL = new EmptyEmail();
         public EmptyEmail() {
```

Comment:
should this be private?
```suggestion
        private EmptyEmail() {
```